charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-apiserver.rules.yaml

```
@@ -49,6 +49,19 @@ groups:
     expr: histogram_quantile(0.5, rate(apiserver_request_duration_seconds_bucket[5m]))
     labels:
       quantile: "0.5"
+  # TODO remove with better metrics in the future (wyb1)
+  - record: shoot:apiserver_watch_duration:quantile
+    expr: histogram_quantile(0.2, sum(rate(apiserver_request_duration_seconds_bucket{verb="WATCH",resource=~"configmaps|deployments|secrets|daemonsets|services|nodes|pods|namespaces|endpoints|events|statefulsets|clusterroles|roles"}[5m])) by (le,scope,resource))
```

In smaller clusters, we might not have enough data for the rate of `5m` to compute a value. Should we increase to `10m`?
```suggestion
    expr: histogram_quantile(0.2, sum(rate(apiserver_request_duration_seconds_bucket{verb="WATCH",resource=~"configmaps|deployments|secrets|daemonsets|services|nodes|pods|namespaces|endpoints|events|statefulsets|clusterroles|roles"}[10m])) by (le,scope,resource))
```

---

I can change it to `10m`. I assumed this is relevant for our large seeds so `5m` would be sufficient.

---

Well, you're right. `5m` should be sufficient. If I think about it: if we have too few values for the rate computation, then we probably also don't run into issues with the watch duration 😄 
So, let's keep `5m`...